dotnet/Simple.Sauce/DriverFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Remote;
+
+namespace Simple.Sauce
+{
+    public class DriverFactory : IDriverFactory
+    {
+        private IWebDriver _driver;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_driver' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4748224)